src/native/libs/configure.cmake

```diff
@@ -121,6 +121,18 @@ check_c_source_compiles(
     "
     HAVE_FLOCK64)
 
+check_c_source_compiles(
```

Comment:
nit: we can use `check_struct_has_member` for these 'HAVE_STRUCTX' checks (see line 304 onwards).